src/styles/blocks/card.scss

```diff
@@ -0,0 +1,83 @@
+@use "../utils/functions";
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  &__product {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+  &__title {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+    margin: 40px 16px 0;
+  }
+  &__code {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: $font-weight-normal;
+    line-height: 14px;
+    text-align: left;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+    color: #616070;
+    margin: 4px 84px 0 16px;
+  }
+  &__buyButton {
+    font-family: Roboto, sans-serif;
+    font-size: 14px;
+    font-weight: $font-weight-bold;
```

Comment:
The variable `$font-weight-bold` is used here, but it is not defined within this file or imported from another file. Ensure that this variable is defined in the `functions` file or any other imported file, or replace it with a valid font-weight value.